test/e2e/built-in-routes.test.js

```diff
@@ -0,0 +1,374 @@
+"use strict";
+
+const webpack = require("webpack");
+const Server = require("../../lib/Server");
+const config = require("../fixtures/client-config/webpack.config");
+const multiConfig = require("../fixtures/multi-public-path-config/webpack.config");
+const runBrowser = require("../helpers/run-browser");
+const port = require("../ports-map")["universal-compiler"];
+
+describe("Built in routes", () => {
+  describe("with simple config", () => {
+    let compiler;
+    let server;
+    let page;
+    let browser;
+    let pageErrors;
+    let consoleMessages;
+
+    beforeEach(async () => {
+      compiler = webpack(config);
+      server = new Server({ port }, compiler);
+
+      await server.start();
+
+      ({ page, browser } = await runBrowser());
+
+      pageErrors = [];
+      consoleMessages = [];
+    });
+
+    afterEach(async () => {
+      await browser.close();
+      await server.stop();
+    });
```

Comment:
This is the right way, we should rewrite all our test on this, it is better for debug and testing, it was in my roadmap, so feel free to do it